client/webserver/webserver_test.go

```diff
@@ -312,6 +313,57 @@ func TestMain(m *testing.M) {
 	os.Exit(doIt())
 }
 
+func TestConnect(t *testing.T) {
+	tCtx, shutdown := context.WithCancel(context.Background())
+	defer shutdown()
+	s, _, _ := newTServer(t, false)
+
+	sWithError, _, _ := newTServer(t, false)
+	sWithError.addr = "bad"
+	type args struct {
+		s *WebServer
+	}
+	tests := []struct {
+		name          string
+		args          args
+		wantErr       bool
+		shutdown      bool
+		expectedError string
+	}{
+		{
+			"start web listener",
+			args{
+				s,
+			},
+			true,
```

Comment:
We don't want an error here, correct? I don't think it is erroring. It's not for me anyway.

Reply:
Good catch.